src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -32,6 +33,21 @@ object LocalBackend {
       writer.write("\n")
     }
   }
+
+  /**
+   * {{{cromwell-executions + workflow.name + workflow.id = cromwell-executions/three-step/0f00-ba4}}}
+   */
```

Comment:
From a test that I was writing I needed to be able to know where the CWD of a task was going to be so I could write files there.  This was to test something like `Array[File] files = ["file1", "file2", "file3"]` in a WDL file